migrations/20221115082230_add_email_column_identities.up.sql

```diff
@@ -0,0 +1,12 @@
+update
+  auth.identities as identities
+set
+  identity_data = identity_data || jsonb_build_object('email', (select email from auth.users where id = identities.user_id))
```

Comment:
probably also good to set the `updated_at` value to a hard-coded timestamp like just the date so we can easily track and rollback the affected rows if this fails on some projects